### ISSUE TITLE
Https auto redirect 

### ISSUE THREAD

**Issue by cyian-1756** [RipMeApp/ripme#1081]
There should be a config option that if true causes all requests to sites that support https to be over https

**Comment by buzzlightmonth** [comment]
I feel like this should rather be the default: the downloader should get all URLs as https, since I'm willing to bet all supported website support https.

**Comment by cyian-1756** [comment]
@buzzlightmonth 

> since I'm willing to bet all supported website support https.

Nope. A lot of the sites supported by ripme either don't have https or if they do it is misconfigured (Like xhamster where they use an invalid cert for some of their cdns)
